DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/_ActivityGraphDataPointError.cshtml

```diff
@@ -0,0 +1,4 @@
+﻿<div class="nhsuk-inset-text nhsuk-u-margin-top-0 nhsuk-u-margin-bottom-4">
+  <span class="nhsuk-u-visually-hidden">Information: </span>
+  <p>Too many chart data points to display. Please select a smaller date range or a bigger reporting interval to view data on a chart.</p>

```

Comment:
Sorry, I should have been more specific maybe. The "inset" component would be a good partial / VC to have, but we should pass the text in in a viewmodel so that this partial isn't specific to the graph data point error, and can be used in other places. Can the inset have different levels (like `information`, `warning`, `error` kind of thing)? If so we could make that parameterizable as well.